src/http/index.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+
+async function axiosRequest(url, method, data) {
```

Comment:
Good practice, because now if you ever want to change functionality to your requests across the board - For example, you needed to change the error handling - you only need to change it in this one file. :clap: